spec/api-utility-process-spec.ts

```diff
@@ -129,6 +129,22 @@ describe('utilityProcess module', () => {
       expect(code).to.equal(exitCode);
     });
 
+    it('does not run JS after process.exit is called', async () => {
+      const file = path.join(os.tmpdir(), `no-js-after-exit-log-${Math.random()}`);
+      const child = utilityProcess.fork(path.join(fixturesPath, 'no-js-after-exit.js'), [`--testPath=${file}`]);
+      const [code] = await once(child, 'exit');
+      expect(code).to.equal(1);
+      let handle = null;
+      try {
+        handle = await fs.open(file);
+        for await (const line of handle.readLines()) {
+          expect(line).to.not.contain('after exit');
+        }
+      } finally {
+        await handle?.close();
+      }
```

Comment:
```suggestion
      const lines = [];
      try {
        handle = await fs.open(file);
        for await (const line of handle.readLines()) {
          lines.push(line);
        }
      } finally {
        await handle?.close();
        await fs.rm(file, { force: true });
      }
      expect(lines.length).to.equal(1);
      expect(lines[0]).to.equal('before exit');
```

Let's ensure we always do the `expect` calls even if there's an error thrown or the file is empty, otherwise we might miss a regression. Also added an `fs.rm` to clean up the temp file.